mod.d/speculativeexecution.yaml

```diff
@@ -0,0 +1,196 @@
+# Copyright 2016-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+--- !ec2rlcore.module.Module
+# Module document. Translates directly into an almost-complete Module object
+name: !!str speculativeexecution
+path: !!str
+version: !!str 1.0
+title: !!str Determine status of Speculative Execution mitigations
+helptext: !!str |
+  Determine status of Speculative Execution mitigations.
```

Comment:
It may be helpful to call out spectrev2 and meltdown here. At a glance, this looks like it may also check spectrev1 and spec_store_bypass.

Reply:
This is intended more for understanding the potential performance impact of these being enabled rather than caring about the specific vulnerabilities. This module is specifically *not* about checking whether or not you are secure, as is called out in description.